sorc/emcsfc_ice_blend.fd/emcsfc_ice_blend.f90

```diff
@@ -97,8 +98,8 @@ program emcsfc_ice_blend
 
  character(len=200)     :: infile, outfile
 
- integer, parameter     :: imax=4320
- integer, parameter     :: jmax=2160
+ integer, parameter     :: imax=4320 !< Constant value on i dim.
+ integer, parameter     :: jmax=2160 !< Constant value on j dim.
```

Comment:
You need to transfer all the documentation. So the descriptions of the other vars, not just imax and jmax.

Reply:
How can I transfer documentation if there is no existing to work with? Please give me an example using emcsfc_ice_blend.f90.